### PR TITLE
DGX doc

### PR DESCRIPTION
**Pull request by tikr7**

### CODE REVIEW

File: DGX/dgx-install-spark.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```
+echo "export SPARK_HOME=$HOME/spark" >> ~/.profile
+source ~/.profile
+echo "export PATH=$PATH:$SPARK_HOME/bin:$SPARK_HOME/sbin" >> ~/.profile
+echo "export PYSPARK_PYTHON=/usr/bin/python3" >> ~/.profile
+source ~/.profile
+```
+- Lastly, install the according python package with pip:
+```
+pip install --user pyspark==3.3.0
```
Comment (ncbkr) [inline DGX/dgx-install-spark.md:37]: Ich würde hier nach best practice zunächst ein venv erzeugen

```sh
python3 -m venv venv
source venv/bin/activate
pip install pyspark==3.3.0
```

File: DGX/dgx-spark-rapids-gpu.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+## Configure your SparkSession accordingly
+
+For more info and details look into [On-Prem](https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-on-prem.html) doc. To be more specific, we use the [Local Mode](https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-on-prem.html#local-mode).
+
+Add at least the following config to your SparkSession:
+```
+    .config("spark.plugins", "com.nvidia.spark.SQLPlugin") \
+    .config("spark.worker.resource.gpu.discoveryScript", f"{HOME}/spark/sparkRapidsPlugin/getGpusResources.sh) \
```
Comment (ncbkr) [inline DGX/dgx-spark-rapids-gpu.md:32]: closing `"` missing at the end of the line. has to be
```
.config("spark.worker.resource.gpu.discoveryScript", f"{HOME}/spark/sparkRapidsPlugin/getGpusResources.sh") \
```

File: DGX/dgx-spark-rapids-gpu.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    .config("jars", f"{HOME}/spark/jars/rapids-4-spark_2.12-22.10.0.jar") \
+```
+
+All configs can be found in the official [doc](https://nvidia.github.io/spark-rapids/docs/configs.html).
+
+## GPU usage
```
Comment (ncbkr) [inline DGX/dgx-spark-rapids-gpu.md:38]: is there an example one can run which utilizes GPU? I understood that not every operation can be executed on GPU

**Review by ncbkr — APPROVED** [review]
some minor things. Found a typo in the gpu spark config which needs to be fixed. the other things are nice to haves

File: DGX/dgx-spark-rapids-gpu.md (hunk 2 above)
Comment (tikr7) [reply DGX/dgx-spark-rapids-gpu.md:32]: Thank you, fixed!

File: DGX/dgx-spark-rapids-gpu.md (hunk 3 above)
Comment (tikr7) [reply DGX/dgx-spark-rapids-gpu.md:38]: This is not exactly clear to me whats executed on CPU and GPU. But all examples I used, produced GPU stages in the DAG.

File: DGX/dgx-install-spark.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+pyspark
+version 3.3.0
+exit()
+```
+
+### Dependency management
+
+We recommend at least S3 dependencies.
+```
+wget https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.3.2/hadoop-aws-3.3.2.jar -O $HOME/spark/jars/hadoop-aws-3.3.2.jar
```
[the remark this answers is not in the document]
Comment (tikr7) [reply DGX/dgx-install-spark.md:52]: hadoop-aws-??? depends on hadoop and not spark version. I added a new section with more details.

File: DGX/dgx-create-sparksession.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+There is no automatic IRSA S3 authorization. Instead you need to use S3 access-key and secret-key like:
+```
+    .config("spark.jars", f"{HOME}/spark/jars/hadoop-aws-3.3.2.jar,{HOME}/spark/jars/aws-java-sdk-bundle-1.11.1026.jar") \
+    .config("spark.hadoop.fs.s3a.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem") \
+    .config("spark.hadoop.fs.s3a.access.key", "*******") \
+    .config("spark.hadoop.fs.s3a.secret.key", "**************") \
+```
+
+## Accessing Spark UI
```
[the remark this answers is not in the document]
Comment (tikr7) [reply DGX/dgx-create-sparksession.md:85]: Ok, I added new doc for a Spark History Server. I made it more generic.

File: DGX/dgx-install-spark.md (hunk 1 above)
Comment (tikr7) [reply DGX/dgx-install-spark.md:39]: Done by @salil-at

**Pull request merged**